ambry-store/src/main/java/com/github/ambry/store/BlobStoreCompactor.java

```diff
@@ -557,6 +579,16 @@ private boolean copyDataByIndexSegment(LogSegment logSegmentToCopy, IndexSegment
     return copiedAll;
   }
 
+  /**
+   * Determines if {@code copyCandidate} belongs to the DELETED_IN_PROGRESS {@link Container}s.
```

Comment:
Determines if {@code copyCandidate} container in the status of  DELETED_IN_PROGRESS or INACTIVE.

Reply:
Updated.